src/rules/rules.js

```diff
@@ -0,0 +1,71 @@
+class Rules{
+   constructor(mode){
+      this.renderPeopleHeader(mode)
```

Comment:
I don't recommend omitting semicolons, it can sometimes have unforeseen consequences, know that they aren't necessary and we do not have a defined codestyle (needs to be changed), but for safety it's better to add semicolons where they should be: https://dev.to/adriennemiller/semicolons-in-javascript-to-use-or-not-to-use-2nli

Reply:
I wasn't even aware of ASI. I've pushed the corrected code